test/libsolidity/util/BytesUtils.cpp

```diff
@@ -198,3 +198,26 @@ string BytesUtils::formatString(bytes const& _bytes)
 
 	return os.str();
 }
+
+string BytesUtils::formatRawBytes(bytes const& _bytes)
+{
+	if (!_bytes.empty())
```

Comment:
if you reverse the `if` conditions/bodies, you saved one indent level, e.g. `if(bytes.empty()) return "[]";`